packages/svelte/src/compiler/phases/2-analyze/index.js

```diff
@@ -1244,6 +1244,10 @@ const common_visitors = {
 				context.state.expression.metadata.dynamic = true;
 			}
 
+			const is_exported = binding.references.some((reference) =>
+				reference.path.some((path_item) => path_item.type === 'ExportSpecifier')
+			);
```

Comment:
this isn't quite right. it will cause _all_ references to have their warnings ignored, not just the export